src/main/webapp/scholarx/2020/styles.css

```diff
@@ -1 +1,31 @@
 /*Add custom styles for the scholarx 2020 page here*/
+ul.timeline {
```

Comment:
This may affect other unordered lists as well. 
Wrap with another class

Reply:
ul.timeline wont affect other lists right?

Reply:
I guess `ul.timeline` could be replaced with just `timeline`

Reply:
ok its fixed what about other things

Reply:
I guess `ul.timeline` could be replaced with just `timeline`